yarn.lock

```diff
@@ -2,6 +2,19 @@
 # yarn lockfile v1
 
 
+"@pangolindex/sdk@10.0.12":
+  version "10.0.12"

```

Comment:
there is some issue here, version is not proper i think

Reply:
done ✅